tailnet/conn.go

```diff
@@ -774,3 +779,15 @@ func Logger(logger slog.Logger) tslogger.Logf {
 		logger.Debug(context.Background(), fmt.Sprintf(format, args...))
 	})
 }
+
+func endpointsEqual(x, y []tailcfg.Endpoint) bool {
```

Comment:
Could maybe just use https://pkg.go.dev/golang.org/x/exp/slices#Equal

Reply:
I just copied how TS does it... I'd rather keep for now.